src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2043,7 +2046,7 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
             if (! validation_manifest.lines().empty())
             {
                 std::string s;
-                s.reserve (188);
+                s.reserve (288);
```

Comment:
I'm not fond of the magic numbers here and in OverlayImpl.cpp, but I can't think of a better alternative.  And the cost if the number becomes wrong is relatively low.